lang/en/tool_s3logs.php

```diff
@@ -48,6 +48,8 @@
 $string['notconfigured'] = 'Missing configuration.';
 $string['prefix'] = 'Log file prefix';
 $string['prefix_desc'] = 'The prefix applied to the uploaded log filename.';
+$string['sdkcredsok'] = 'AWS credentials found. This setting can be safely enabled.';
+$string['sdkcredserror'] = 'Couldn\'t find AWS credentials. It\'s unsafe to enable this setting. Follow up <a href="https://docs.aws.amazon.com/sdk-for-php/v3/developer-guide/guide_credentials.html">AWS documentation</a>.';
```

Comment:
Doesn't codechecker complain about the string length?

Reply:
you can see all green soooo....

Reply:
I think it's just a warning in lang string file. So should be ok.